flipt-client-java/README.md

```diff
@@ -124,6 +124,13 @@ The `FliptEvaluationClient` supports the following authentication strategies:
 - [Client Token Authentication](https://docs.flipt.io/authentication/using-tokens)
 - [JWT Authentication](https://docs.flipt.io/authentication/using-jwts)
 
+### Error Strategies
+
+The client supports the following error strategies:
```

Comment:
Would you mind adding the error strategy method description to the builder section above too please?